src/main/resources/anniv6Resources/localization/eng/Storm/Powerstrings.json

```
@@ -0,0 +1,6 @@
+{
+  "${ModID}:ConduitPower": {
+    "NAME": "Conduit",
+    "DESCRIPTIONS": ["Target will receive #b", " blockable damage at the end of the turn."]
```

```suggestion
    "DESCRIPTIONS": ["Target will receive #b", " blockable damage at the end of the turn."]
```
Not sure if this is the best wording here, end of turn would not be correct if this is on the player, since it happens later. End of round might also be bad, so after the monsters' turn?